test/image-blueprints/layer1-base/group2/rhel94-microshift-yminus2.toml

```diff
@@ -0,0 +1,55 @@
+name = "rhel-9.4-microshift-4.{{ .Env.YMINUS2_MINOR_VERSION }}"
```

Comment:
Shouldn't this be 9.3?
4.15 is not supported on 9.4, but since bootc is only participating in the upgrade image (`9.3 + 4.15 -> bootc 9.4 + 4.17`), then it should be fine?

Reply:
There's a problem with 9.3 OS because it does not container `bootc` RPM, so we have to start from 9.4
I guess, that answers the question, so I'd better remove this image at all?

Reply:
I think there's no point in testing unsupported scenario